packages/components/src/components/grid/sizesTransformation.tsx

```diff
@@ -0,0 +1,63 @@
+import { sizableProps, SizedProp, Sizes } from './grid.interfaces';
+
+export function propsToSizesArray(sizes: string[]): Sizes[] {
+  return [
+    { columnName: 'sm', size: sizes[0] },
+    { columnName: 'md', size: sizes[1] },
+    { columnName: 'lg', size: sizes[2] },
+    { columnName: 'xl', size: sizes[3] },
+    { columnName: 'xxl', size: sizes[4] },
+  ];
+}
+
+export function isSizesEmpty(sizes: Sizes[]): boolean {
+  let isEmpty = true;
+  for (const size of sizes) {
+    if (size.size) {
+      isEmpty = false;
+      break;
+    }
+  }
+  return isEmpty;
+}
```

Comment:
And also the rest of the file, here for instance `isValuesEmpty` and `values`, and so on…